flyway/run-app.sh

```diff
@@ -49,6 +49,7 @@ run_test "TestYBLocking.advisoryLockTest" "flyway/start.sh"
 
 sed -i '$ s/,$//' temp_report.json # Remove trailing comma from the last JSON object
 echo "]" >> temp_report.json
+sed -i 's/\t/    /g' temp_report.json # Replace tabs with spaces
```

Comment:
Is this required in scripts of other drivers/tools as well?

Reply:
I think, yes. Seen some online json validators rejecting the json text if it contained tabs or new lines.